### DISCUSSION TITLE
photoprism installation fail

### DISCUSSION THREAD

**Discussion by JF62** [tteck/Proxmox#1996] [Help Needed] [answered]
[ERROR] in line 40: exit code 1: while executing command cat <<EOF > /opt/photoprism/config/.env
PHOTOPRISM_AUTH_MODE='password'
PHOTOPRISM_ADMIN_PASSWORD='changeme'
PHOTOPRISM_HTTP_HOST='0.0.0.0'
PHOTOPRISM_HTTP_PORT='2342'
PHOTOPRISM_SITE_CAPTION='https://tteck.github.io/Proxmox/'
PHOTOPRISM_STORAGE_PATH='/opt/photoprism/storage'
PHOTOPRISM_ORIGINALS_PATH='/opt/photoprism/photos/originals'
PHOTOPRISM_IMPORT_PATH='/opt/photoprism/photos/import'
EOF

++ [[ 40 -eq 24 ]]
+ exit

looks like the conf directory aren't created forehand

**Comment by tteck** [comment, marked as answer]
Fixed
